resources/kcp/charts/kyma-environment-broker/templates/subscription-cleanup-job.yaml

```diff
@@ -16,7 +16,7 @@ spec:
             - image: "{{ .Values.global.images.containerRegistry.path }}/{{ .Values.global.images.kyma_environments_subscription_cleanup_job.dir }}kyma-environment-subscription-cleanup-job:{{ .Values.global.images.kyma_environments_subscription_cleanup_job.version }}"
               name: subscription-cleanup-job
               command:
-                - "/bin/subscriptioncleanup"
+                - /bin/main
```

Comment:
Will break the job. You changes should not affect jobs other than environment cleanup (according to what you said).